packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -197,6 +198,16 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
    */
   public readonly asg: AutoScalingGroup;
 
+  /**
+   * The version of Deadline that the RenderQueue uses
+   */
+  public readonly version: IVersion;
+
+  /**
+   * Whether SEP policies have been added
+   */
+  private addedSEPPolicies: boolean = false;
```

Comment:
This is *very* nitpicky, but how about renaming this to `haveAddedSEPPolicies`?

Reply:
Sure!